examples/vm_scheduling/reinforcement_learning/data_scripts/create_new_data.py

```diff
@@ -0,0 +1,66 @@
+import pandas as pd
```

Comment:
Add comments for this file, used for what ?

Or you can add an README for this whole example, to introduce what's the files in this folder, what's the steps (generating data, run ...)